package.json

```diff
@@ -17,7 +17,7 @@
     "@docusaurus/core": "^3.2.0",
     "@docusaurus/plugin-client-redirects": "^3.2.0",
     "@docusaurus/preset-classic": "^3.2.0",
-    "@docusaurus/theme-common": "^3.0.1",
+    "@docusaurus/theme-common": "^3.2.0",
```

Comment:
Is it right to change also these files?

Reply:
Good catch. I think it is ok because things are still working and my env requires that to make docs work